Description: Fix failing PairQueue.Ordering test.
Origin: https://github.com/Macaulay2/M2/commit/9b010dc
Last-Update: 2015-03-19

--- a/src/test/PairQueue.cpp
+++ b/src/test/PairQueue.cpp
@@ -114,6 +114,7 @@
   // distinguish all pairs and that, according to the pairdata,
   //   (11,0) < (11,10) = (111,0) < (11,5)
   // so the order that pairs are extracted mix up columns 11 and 111.
+
   for (size_t col = 0; col < 112; ++col) {
 	Index const* begin = 0;
 	Index const* end = 0;
@@ -121,20 +122,24 @@
 	  Index const rows[] = {0};
 	  begin = rows;
 	  end = rows + sizeof(rows) / sizeof(rows[0]);
+      pq.addColumnDescending(begin, end);
 	} else if (col == 11) {
 	  Index const rows[] = {0, 10, 5};
 	  begin = rows;
 	  end = rows + sizeof(rows) / sizeof(rows[0]);
+      pq.addColumnDescending(begin, end);
 	} else if (col == 13) {
 	  Index const rows[] = {12, 3, 7};
 	  begin = rows;
 	  end = rows + sizeof(rows) / sizeof(rows[0]);
+      pq.addColumnDescending(begin, end);
 	} else if (col == 111) {
 	  Index const rows[] = {0, 100};
 	  begin = rows;
 	  end = rows + sizeof(rows) / sizeof(rows[0]);
-	}
-	pq.addColumnDescending(begin, end);
+      pq.addColumnDescending(begin, end);
+	} else
+      pq.addColumnDescending(begin, end);
   }
   std::ostringstream out;
   std::string lastPd;
